### DISCUSSION TITLE
旁路由，自定义路由模式，无法打开ipv6网站。

### DISCUSSION THREAD

**Discussion by zorgzola** [immortalwrt/homeproxy#209] [Q&A]
如题，自定义路由模式，遇到关于ip地址解析的路由规则、DNS等的都设为仅ipv4，ipv6支持也打过钩。就是不能连接ipv6的网站。
homeproxy设为大陆白名单模式就可以，用passwall也可以。
请大家帮看看，要怎么设置，谢谢。

**Comment by 1715173329** [comment]
本项目不支持所谓“旁路由”，免问即可。
